api/src/main/java/com/debugmafia/clueless/state/Game.java

```diff
@@ -0,0 +1,144 @@
+package com.debugmafia.clueless.state;
+
+import com.debugmafia.clueless.actions.*;
+import com.debugmafia.clueless.model.*;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.*;
+
+
+import javax.tools.DocumentationTool.Location;
+
+public class Game {
+    private HashSet<Card> winningCards;
+    private Board board;
+    private Turn turnInstance;
+    private List<Player> playerList;
+
+    public Game(Set<Player> players) {
+        this.board = new Board();
+        this.playerList = new ArrayList<>(players);
+        //TODO: Conversion above needs to have Mrs. Scarlett first
+        //TODO: Set the initialPlayers private List to the players
+        //TODO: Set the game state to GameState.IN_PROGRESS
+
+        this.winningCards = new HashSet<>(this.board.drawWinningCards());
```

Comment:
I think `drawWinningCards()` already returns a `Set`? Which means you don't have to pass this into a `HashSet` instance.

Reply:
It does, but it yelled at me without that explicit conversion. Though it could have been something else.

Reply:
That's interesting. Okay, maybe it's fine as is then.